cmd/studio.go

```diff
@@ -0,0 +1,39 @@
+package cmd
+
+import (
+	"fmt"
+	"os/exec"
+	"runtime"
+
+	"github.com/spf13/cobra"
+	"github.com/supabase/cli/internal/utils"
+)
+
+var studioCmd = &cobra.Command{
+	Use:   "studio",
+	Short: "Opens Supabase Studio in your browser",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		url := utils.GetSupabaseDashboardURL()
+		var err error
+
+		switch runtime.GOOS {
```

Comment:
Let's use the `RunOpenCmd` function to handle this more reliably on Linux and Windows.

- https://github.com/supabase/cli/blob/develop/internal/login/login_linux.go
- https://github.com/supabase/cli/blob/develop/internal/login/login_windows.go